saltproc/tests/test_saltproc.py

```diff
@@ -1,110 +0,0 @@
-from __future__ import absolute_import, division, print_function
```

Comment:
why is this whole file of tests deleted? Is it because this PR has removed all of this functionality? Has equivalent functionality replaced this? Will previous simulations run properly?